sdk/src/main/assets/hcaptcha-form.html

```diff
@@ -14,6 +14,10 @@
         #hcaptcha-container {
             margin-top: 5px;
         }
+        /* overwrite hCaptcha iframe overlay which adds a #FFF background with opacity 0.05 */
+        div > div:nth-child(2) {
```

Comment:
@brdlyptrs what do you think about adding some class to overlay? `.hcaptcha-overlay` for example

Reply:
@DSergiu this change overrides both mobile and desktop view. Is this intentional?

Reply:
1. Having a class for the selector would have been nice
2. Yes, overlay must be removed for all screen sizes to be consistent with the `loading` state